appinventor/appengine/src/com/google/appinventor/client/youngandroid/TextValidators.java

```diff
@@ -77,6 +77,7 @@ public static boolean checkNewProjectName(String projectName) {
   public static boolean checkNewComponentName(String componentName) {
 
     // Check that it meets the formatting requirements.
+    
```

Comment:
Not sure why this blank line was added. The comment above goes with this if statement and so it doesn't make sense to add visual separation.